tst/EnergyPlus/unit/AdvancedAFN.unit.cc

```diff
@@ -85,28 +85,28 @@ TEST_F(EnergyPlusFixture, AirflowNetwork_AdvancedTest_Test1)
     int CurveNum;
 
     AirflowNetworkNumOfOccuVentCtrls = 1;
-    state.dataAirflowNetworkBalanceManager->OccupantVentilationControl.allocate(AirflowNetworkNumOfOccuVentCtrls);
-    state.dataAirflowNetworkBalanceManager->OccupantVentilationControl(1).MinOpeningTime = 4;
-    state.dataAirflowNetworkBalanceManager->OccupantVentilationControl(1).MinClosingTime = 4;
-    state.dataAirflowNetworkBalanceManager->OccupantVentilationControl(1).MinTimeControlOnly = true;
+    state->dataAirflowNetworkBalanceManager->OccupantVentilationControl.allocate(AirflowNetworkNumOfOccuVentCtrls);
+    state->dataAirflowNetworkBalanceManager->OccupantVentilationControl(1).MinOpeningTime = 4;
+    state->dataAirflowNetworkBalanceManager->OccupantVentilationControl(1).MinClosingTime = 4;
+    state->dataAirflowNetworkBalanceManager->OccupantVentilationControl(1).MinTimeControlOnly = true;
 
     TimeOpenElapsed = 3.0;
     TimeCloseElapsed = 0.0;
-    state.dataAirflowNetworkBalanceManager->OccupantVentilationControl(1).calc(state, 1, TimeOpenElapsed, TimeCloseElapsed, OpenStatus, OpenProbStatus, CloseProbStatus);
+    state->dataAirflowNetworkBalanceManager->OccupantVentilationControl(1).calc(*state, 1, TimeOpenElapsed, TimeCloseElapsed, OpenStatus, OpenProbStatus, CloseProbStatus);
```

Comment:
And here's the other change you'll see many, many times.  The guts of EnergyPlus expect a state instance to be passed in (technically a reference to it), so you just need to dereference the pointer with an asterisk and that's it.